src/content/en/progressive-web-apps/checklist.md

```diff
@@ -715,7 +715,7 @@ Note: This check list only applies if notifications implemented
 <table class="responsive">
   <tbody>
     <tr>
-      <th colspan=2><h3>Ensure that the UI that encourages users to turn on Push Notifications isn't overly aggressive.</h3></th>
+      <th colspan=2><h3>Ensure that the UI that encourages users to turn on Push Notifications isn't overly aggressive</h3></th>
```

Comment:
Can you shorten this title. Maybe something like 'Avoid aggressive permission requests for notifications'. I don't really care how it's phrased. It just should be shorter.